src/extension/inpage-script/index.js

```diff
@@ -121,6 +121,27 @@ if (document) {
     },
     { capture: true }
   );
+  // Listen for webln events from the extension
+  // emit events to the websites
+  window.addEventListener("message", (event) => {
+    if (event.source === window && event.data.action === "accountChanged") {
+      if (
+        (window.webln && window.webln.enabled) ||
+        (window.nostr && window.nostr.enabled)
+      )
+        eventEmitter(event.data.action, event.data.scope);
+    }
+  });
 } else {
   console.warn("Failed to inject WebLN provider");
 }
+
+function eventEmitter(action, scope) {
+  if (scope == "nostr") {
```

Comment:
could it be simpler like this? then we don't need to add more checks for e.g. window.webbtc or window.alby
```
if (window[scope]) {
  window[scope].emit(action);
}
```

Reply:
cool! i tried this earlier but didn't worked in console so avoided it. but works well enough in code. thanks for this suggestion